JS/script.js

```diff
@@ -14,4 +14,39 @@ function generateCanvas(row, column) {
 
     document.querySelector('.canvas').innerHTML = canvas
 
-}
\ No newline at end of file
+}
+
+document.querySelector('html').addEventListener('mousedown', function () {
+    clickdown = true
+})
+
+document.querySelector('html').addEventListener('mouseup', function () {
```

Comment:
suggest adding a comment in for each of these global events explaining why they are needed.